man/systemd.network.xml

```diff
@@ -1377,6 +1377,16 @@
             service type to CS6 (network control) or CS4 (Realtime). Defaults to CS6.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>MultiPathRoute=<replaceable>address</replaceable> [dev=<replaceable>name</replaceable>] [weight=<replaceable>number</replaceable>]</varname></term>
+          <listitem>
+            <para>Configures multipath route. Takes gateway address. Optionally, takes a network
+            interface name or index prefixed with <literal>dev=</literal>, and a weight in 1..256
```

Comment:
May be add `Multipath routing is the routing technique of using multiple alternative paths through a network.